lib/provider.js

```diff
@@ -29,10 +29,7 @@ const get = (obj, path, defaultValue = undefined) => {
  * @param config
  * @returns {{private_key}|{client_email}|{project_id}|any}
  */
-const checkServiceAccount = (config) => {
-  if (!config.serviceAccount) {
-    throw new Error('"Service Account JSON" is required!');
-  }
+const checkServiceAccount = (config = {}) => {
```

Comment:
Now `config` defaults to empty object. Consequently it won't file with `TypeError`. Instead it will throw the more detailed error about which fields are missing.